### ISSUE TITLE
Participant's audio is unmute but no sound

### ISSUE THREAD

**Issue by cgri1** [jitsi/jitsi-meet#14135]
### Description:

The participant's audio is unmute but no sound

### Steps to reproduce:

1. Participants join a meeting.
2. Noise suppression settings default open.
3. Participants unmute their microphones. 
4. Participant plug-in jack headphones.
5. And then Participant plug-out.
6. The participant's audio is unmute but no sound.

### Server information:

- Jitsi Meet version: latest version
- Operating System: MacOs

### Client information:

- Browser / app version: Chrome (116, 118, 119)
- Operating System: MacOs

**Comment by saghul** [comment]
Does not enabling noise-suppression alter the results in any way?

**Comment by cgri1** [comment]
Yes, If I disable noise suppression settings default, There is no problem.

**Comment by saghul** [comment]
PIng @andrei-gavrilescu

**Comment by andrei-gavrilescu** [comment]
Will check it.

**Comment by cgri1** [comment]
Is there any progress on this problem?

**Comment by andrei-gavrilescu** [comment]
I've managed to reproduce the issue, seems to be happening consistently. I'll add it to the backlog. 
As a short workaround, simply switch the device in the device selection popup.
Thank you for the detailed report!

**Comment by haseeb-neslit** [comment]
I am facing this exact same issue. During the meeting, even though, the participants had their audio unmuted but still sometimes, there's no sound coming from their side. Though, we checked and everything in auto settings / device config was fine.

**Comment by Heinrich4771** [comment]
Hi,

I‘ve got the same issue on my MacMini. I use this setup now for 2 years but since last week it stopped working. The test sound via the jitsi setup menu works, but I can not hear the participants of the meeting. However they hear me. Opening the same meeting on the iPad works just  fine. 

System used: MacMini with macOS Tahoe 26.2 and the Safari browser.

Just tested, with Chrome I do not have this issue.  

Best Regards
Heinrich
